src/patternfly/components/Table/examples/table-compound-expansion-example.hbs

```diff
@@ -57,22 +57,22 @@
     {{/table-tr}}
 
     {{#> table-tr table-tr--expandable="true" table-tr--IsExpanded="true"}}
-      {{#> table-td table-td--attribute='colspan="7"'}}
-        {{#> table-nested table--id=(concat table--id '-nested-table-1')}}
+      {{#> table-td table-td--attribute='colspan="7"' table-td--modifier="pf-m-no-padding"}}
+        {{#> table-nested table--id=(concat table--id '-nested-table-1"')}}
```

Comment:
need to remove the extra double quotes in all of these nested table `table--id`'s. those were fixed in https://github.com/patternfly/patternfly-next/pull/1614/files

```suggestion
        {{#> table-nested table--id=(concat table--id '-nested-table-1')}}
```